src/pages/pt-BR/guides/markdown-content.md

```diff
@@ -294,24 +314,24 @@ Você pode importar o [componente Markdown nativo do Astro](/pt-BR/reference/api
 import { Markdown } from 'astro/components';
 import Layout from '../layouts/Layout.astro';
 
-const expressions = 'Lorem ipsum';
+const expressões = 'Lorem ipsum';
```

Comment:
```suggestion
const expressoes = 'Lorem ipsum';
```

Reply:
I'll leave it as `expressões` just because it's used below to make the sentence `{expressões} jsx`. It's not recommended, but it is technically, valid JS.